buildpacks/npm/shpec/build_shpec.sh

```diff
@@ -13,355 +13,354 @@ source "${shpec_dir}/../lib/utils/log.sh"
-			end
-		end
+describe "package-lock.json checksum matches the cached checksum"
+it "reuses the node_modules cache"
+#todo
+end
+end
 
-		rm_temp_dirs "$project_dir" "$layers_dir"
-	end
+rm_temp_dirs "$project_dir" "$layers_dir"
+end
 
-	describe "run_build"
-		project_dir=$(create_temp_project_dir)
+describe "run_build"
+project_dir=$(create_temp_project_dir)
 
-		use_npm 6
+use_npm 6
 
-		create_temp_package_json "$project_dir" "/scripts/heroku-postbuild"
+create_temp_package_json "$project_dir" "/scripts/heroku-postbuild"
 
-		describe "when package.json has a heroku-postbuild script"
-			it "runs the heroku-postbuild script"
-				run_build "$project_dir"
+describe "when package.json has a heroku-postbuild script"
+it "runs the heroku-postbuild script"
+run_build "$project_dir"
 
-				assert file_present "heroku-postbuild"
-			end
-		end
+assert file_present "heroku-postbuild"
+end
+end
 
-		create_temp_package_json "$project_dir" "/scripts/build"
+create_temp_package_json "$project_dir" "/scripts/build"
 
-		describe "when package.json has a build script"
-			it "runs the build script"
-				run_build "$project_dir"
+describe "when package.json has a build script"
+it "runs the build script"
+run_build "$project_dir"
 
-				assert file_present "build"
-			end
-		end
+assert file_present "build"
+end
+end
 
-		rm_temp_dirs "$project_dir"
-	end
+rm_temp_dirs "$project_dir"
+end
 
-	describe "write_launch_toml"
-		layers_dir=$(create_temp_layer_dir)
-		project_dir=$(create_temp_project_dir)
+describe "write_launch_toml"
+layers_dir=$(create_temp_layer_dir)
+project_dir=$(create_temp_project_dir)
 
-		create_temp_package_json "$project_dir" ""
+create_temp_package_json "$project_dir" ""
 
-		it "does not create a launch.toml file with no start script"
-			assert file_absent "$layers_dir/launch.toml"
+it "does not create a launch.toml file with no start script"
+assert file_absent "$layers_dir/launch.toml"
 
-			write_launch_toml "$project_dir/package.json" "$layers_dir/launch.toml"
+write_launch_toml "$project_dir/package.json" "$layers_dir/launch.toml"
 
-			assert file_absent "$layers_dir/launch.toml"
-		end
+assert file_absent "$layers_dir/launch.toml"
+end
 
-		create_temp_package_json "$project_dir" "/scripts/start"
+create_temp_package_json "$project_dir" "/scripts/start"
 
-		it "creates a launch.toml file"
-			assert file_absent "$layers_dir/launch.toml"
+it "creates a launch.toml file"
+assert file_absent "$layers_dir/launch.toml"
 
-			write_launch_toml "$project_dir/package.json" "$layers_dir/launch.toml"
+write_launch_toml "$project_dir/package.json" "$layers_dir/launch.toml"
 
-			assert file_present "$layers_dir/launch.toml"
-		end
+assert file_present "$layers_dir/launch.toml"
+end
 
-		rm_temp_dirs "$project_dir" "$layers_dir"
-	end
+rm_temp_dirs "$project_dir" "$layers_dir"
+end
 
-	unstub_command "log_info"
-	unstub_command "node"
-	rm_tools_and_mocks
+unstub_command "log_info"
+unstub_command "node"
+rm_tools_and_mocks
```

Comment:
Is this whitespace change intentional? I'm not super familiar with shpec. I find the indented version easier to follow than these flattened ones. I'm approving, if it's intentional, you can resolve this.

Reply:
Unintentional. IDE auto-formatter got ahold of it. I've reverted this in https://github.com/heroku/buildpacks-nodejs/pull/1099/commits/e7c9fe21d614aed2f9735727068b3327aa608dfd and added a new line at the end in https://github.com/heroku/buildpacks-nodejs/pull/1099/commits/51550eeae2617b54dcc916edd5353f587ac16b2f